drivers/display/Kconfig.ssd1306

```diff
@@ -35,9 +35,4 @@ config SSD1306_SH1106_COMPATIBLE
 
 endchoice
 
-config SSD1306_REVERSE_MODE
-	bool "SSD1306 reverse mode"
-	help
-	  SSD1306 reverse video mode.
-
```

Comment:
This commit should be second change before multi instance support commit.

Reply:
I reordered commits.